src/resonanceReconstruction/rmatrix/src/makeLegacyUnresolvedSpinGroups.hpp

```diff
@@ -16,7 +16,10 @@ makeLegacyUnresolvedSpinGroups(
     return legacy::unresolved::SpinGroup(
                Channel< Neutron >( in, in, 0. * electronVolt,
                                    { l, 0.5, endfJValue.spin(),
-                                     std::pow( -1, l ) > 0. ? +1 : -1 }, radii ),
+                                     std::pow( -1, l ) > 0.
```

Comment:
Would writing `L` rather than `l` be appropriate terminology? It's so much less easy to confuse as being a `1`.